jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -401,6 +401,11 @@
           default: '1'
           description: set to 0 to not deploy aodh
 
+      - string:
+          name: want_pci_passthrough
```

Comment:
I think this is used seldomly enough, that you can drop this and use custom_settings='export want_pci_passthrough=1' in the template instead.

Reply:
Sure. I thought it's required to define in here in order to be consumed in [this proposed ci job template](https://github.com/SUSE-Cloud/automation/pull/3543/files#diff-e94211e41968cb76d0ffe7b870db09c9). Did I miss anything?

Reply:
If you want to use its own argument for that, like the PR currently does, you need to define it. If you are fine with passing it through the custom_settings argument then you don't.